migration/postgres/README.md

```diff
@@ -247,9 +245,6 @@ select count(*) from user_role
 count(*)
 107
 
-select count(*) from help_report
-count(*)
-13
```

Comment:
Please revert these changes to this file, it's an historical doc that we should keep around in case we need double check our validation.